Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -21,6 +21,17 @@ Write-Verbose -Verbose "configFile = $configFile"
 Write-Verbose -Verbose "dbConnectionString = $dbUrl"
 Write-Verbose -Verbose "dbUsername = $dbUsername"
 
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
```

Comment:
Is any of this code git-specific? It doesn't look like anything would need to change for code stored in TFVC. If that is the case, you could drop "Git" from the variable name.
Also, "CodeAnalysis" is a very general term: "PullRequestSonarQubeAnalysisEnabled" would make the connection to SonarQube obvious.